runtimes/manta/runtime/src/tests.rs

```diff
@@ -74,17 +74,47 @@ impl ExtBuilder {
 
 #[test]
 fn balances_should_work() {
-	ExtBuilder::default().build().execute_with(|| {
-		// transfer
-		assert_ok!(crate::Balances::transfer(
-			crate::Origin::signed(ALICE),
-			MultiAddress::Id(BOB),
-			20 * MA
-		));
-		// check balance after transfer
-		assert_eq!(crate::System::account(ALICE).data.free, 1000 * MA - 20 * MA);
-		assert_eq!(crate::System::account(BOB).data.free, 1000 * MA + 20 * MA);
-	});
+	ExtBuilder::default()
+		.one_thousand_for_alice_n_bob()
+		.build()
+		.execute_with(|| {
+			// transfer
+			assert_ok!(crate::Balances::transfer(
+				crate::Origin::signed(ALICE),
+				MultiAddress::Id(BOB),
+				20 * MA
+			));
+			// check balance after transfer
+			assert_eq!(crate::System::account(ALICE).data.free, 1000 * MA - 20 * MA);
+			assert_eq!(crate::System::account(BOB).data.free, 1000 * MA + 20 * MA);
+		});
+}
+
+#[test]
+#[ignore = "It looks no way to remove panic while building manta_genesis_config."]
+// https://github.com/paritytech/substrate/blob/v3.0.0/frame/balances/src/lib.rs#L481
+fn check_existential_deposit() {
+	let initial_authorities = vec![(
+		ALICE,
+		ALICE,
+		hex!["9becad03e6dcac03cee07edebca5475314861492cdfc96a2144a67bbe9699332"].unchecked_into(),
+		hex!["6e7e4eb42cbd2e0ab4cae8708ce5509580b8c04d11f6758dbf686d50fe9f9106"].unchecked_into(),
+	)];
+	let root_key = ALICE;
+	let stash = 9 * cMA;
+
+	let init_balances: Vec<(AccountId, Balance)> = vec![(ALICE, 99 * cMA), (BOB, 99 * cMA)];
```

Comment:
here should be existential deposit rather than hard coded.

Reply:
Removed this hardcoded value, just check it again.